vllm/v1/cudagraph_dispatcher.py

```diff
@@ -39,11 +39,24 @@ def __init__(self, vllm_config: VllmConfig):
             CUDAGraphMode.FULL: set(),
         }
 
-        assert not self.cudagraph_mode.requires_piecewise_compilation() or \
-            (self.compilation_config.level == CompilationLevel.PIECEWISE and
-             self.compilation_config.splitting_ops_contain_attention()), \
+        not_use_piecewise_compilation = (
+            not self.cudagraph_mode.requires_piecewise_compilation())
+
+        use_fx_graph_piecewise_compilation = (
+            self.compilation_config.level == CompilationLevel.PIECEWISE
+            and self.compilation_config.splitting_ops_contain_attention())
+
+        use_inductor_piecewise_compilation = (
+            self.compilation_config.use_inductor_graph_partition
+            and not self.compilation_config.splitting_ops_contain_attention())
```

Comment:
Could we rearch this so that we can just `assert self.compilation_config.is_attention_compiled_piecewise()` which contains the detailed checks